CHANGELOG.md

```diff
@@ -17,6 +17,11 @@
 
 * `NavigationMapView.removeAlternativeRoutes()` and `NavigationMapView.removeContinuousAlternativeRoutesDurations()` were made public to provide a way to remove previously shown alternative routes and alternative routes duration annotations, respectively. ([#4134](https://github.com/mapbox/mapbox-navigation-ios/pull/4134))
 
+### Routing
+
+* Route refreshing now respects current user progress on a leg to reduce update size and improve updating longer routes. ([#4111](https://github.com/mapbox/mapbox-navigation-ios/pull/4111))
+* Added `RoutingProvider.refreshRoute(indexedRouteResponse:fromLegAtIndex:currentRouteShapeIndex:currentLegShapeIndex:completionHandler:)` to request a refresh starting from specified shape index. Updated `RouteLegProgress` and `RouteProgress` initializers to include shape indices, and added `RouteProgress.refreshRoute(with:at:legIndex:legShapeIndex:)` to apply partial route refresh. ([#4111](https://github.com/mapbox/mapbox-navigation-ios/pull/4111))
```

Comment:
Not sure what the second part of the first blurb refers to, but the primary benefit is a reduction in data usage. The two blurbs are the same feature, so we can start with the user benefit and move on to the developer-facing change that implements it.

```suggestion
* Reduced the amount of data received when periodically refreshing a long route. Added `RoutingProvider.refreshRoute(indexedRouteResponse:fromLegAtIndex:currentRouteShapeIndex:currentLegShapeIndex:completionHandler:)` to request a refresh starting from specified shape index. Updated `RouteLegProgress` and `RouteProgress` initializers to include shape indices, and added `RouteProgress.refreshRoute(with:at:legIndex:legShapeIndex:)` to apply partial route refresh. ([#4111](https://github.com/mapbox/mapbox-navigation-ios/pull/4111))
```